per/views.py

```diff
@@ -1,70 +1,34 @@
-import json, datetime, pytz
-from django.http import JsonResponse, HttpResponse
-from api.views import (
-    bad_request,
-    bad_http_request,
-    PublicJsonPostView,
-    PublicJsonRequestView,
-)
-from rest_framework import viewsets
-from django.contrib.auth.models import User
+from django.db import transaction
+from django.http import JsonResponse
+from django.utils import timezone
+from rest_framework import authentication, permissions
+from rest_framework.views import APIView
 from .models import (
     Draft, Form, FormData, WorkPlan, Overview
 )
-from rest_framework.authtoken.models import Token
-
-def create_draft(raw):
-    Draft.objects.filter(code=raw['code'], country_id=raw['country_id'], user_id=raw['user_id']).delete()  # If exists (a previous draft), delete it.
-    draft = Draft.objects.create(code       = raw['code'],
-                               user_id      = raw['user_id'],
-                               data         = raw['data'],
-                               country_id   = raw['country_id'],
-                               )
-    draft.save()
-    return draft
-
-def create_form(raw):
-
-    form = Form.objects.create(code         = raw['code'],
-                               name         = raw['name'],
-                               language     = raw['language'],
-                               user_id      = raw['user_id'],
-                               country_id   = raw['country_id'],
-                               ns           = raw['ns'],
-                               ip_address   = raw['ip_address'],
-                               started_at   = raw['started_at'],
-                               ended_at     = raw['ended_at'],
-                               submitted_at = raw['submitted_at'],
-                               comment      = raw['comment'],
-                               validated    = raw['validated'],
-                               finalized    = raw['finalized'],
-                               # unique_id  = raw['unique_id'], # only KoBo form provided
-                               )
-    form.save()
-    return form
-
-def create_form_data(raw, form):
-    form_data = FormData.objects.create(form= form,
-                            question_id     = raw['id'],
-                            selected_option = raw['op'],
-                            notes           = raw['nt'],
-                            )
-    form_data.save()
-    return form_data
-
-class FormSent(PublicJsonPostView):
-    def handle_post(self, request, *args, **kwargs):
```

Comment:
@batpad @thenav56 This didn't need authentication before but I added it now, so thus why the build failed: https://app.circleci.com/pipelines/github/IFRCGo/go-api/613/workflows/4c8458c8-22d6-491e-81e5-7f1e37800d85/jobs/2529

Should I remove the auth from this? It was weird to me why we allowed insertion without auth...